File: data/imported/knowledge/slides/java/merged/snippets/snippet_054.java
@ParameterizedTest(name = "dash style {0} can be set on shape line format")
    @MethodSource("dashStyles")
    void testMultipleDashStyles(LineDashStyle style) {
        try (var pres = new Presentation()) {
            var slide = pres.getSlides().get(0);
            var shape = slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 50);
            shape.getLineFormat().setDashStyle(style);
            assertThat(shape.getLineFormat().getDashStyle()).isEqualTo(style);
        }
    }

File: data/imported/knowledge/slides/cpp/merged/snippets/snippet_054.cpp
TEST_F(ShapesIntegrationTest, ClearShapes) {
    Presentation pres;
    auto& slide = pres.slides()[0];
    slide.shapes().add_auto_shape(ShapeType::RECTANGLE, 50, 50, 200, 100);
    slide.shapes().clear();
    EXPECT_EQ(slide.shapes().size(), 0);
}